### PR TITLE
Add get endpoint for flow mirror points

### PR DESCRIPTION
**Pull request by dpoltavets**
Related to: #3849

**MERGE NOTE: DO NOT MERGE BEFORE MERGING #4233**

### CODE REVIEW

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java [hunk 1]
```diff
@@ -540,4 +543,11 @@ public MirrorPointDirection mapMirrorPointDirection(String mirrorPointDirection)
     @Mapping(target = "sinkEndpoint", expression = "java(new FlowEndpointV2(response.getSinkEndpointSwitchId(), "
             + "response.getSinkEndpointPort(), response.getSinkEndpointVlan(), response.getSinkEndpointInnerVlan()))")
     public abstract FlowMirrorPointResponseV2 toFlowMirrorPointResponseV2(FlowMirrorPointResponse response);
+
+    public abstract FlowMirrorPointsResponseV2 toFlowMirrorPointsResponseV2(FlowMirrorPointsDumpResponse response);
+
+    @Mapping(target = "sinkEndpoint", expression = "java(new FlowEndpointV2("
```
Comment (surabujin) [inline src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java:549]: Do not embed java code.

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/error/CacheException.java [hunk 2]
```diff
@@ -54,6 +54,21 @@ public CacheException(ErrorType errorType, String errorMessage, String errorDesc
...
 
+    /**
+     * Instance constructor.
+     *
+     * @param errorType        error type
+     * @param errorMessage     error message
+     * @param errorDescription error description
+     * @param throwable        error catch
+     */
+    public CacheException(ErrorType errorType, String errorMessage, String errorDescription, Throwable throwable) {
```
Comment (surabujin) [inline src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/error/CacheException.java:65]: Why for we need it? Are you going to serialize traceback object and transfer it between kilda services?

**Review by surabujin — CHANGES_REQUESTED** [review]

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java (hunk 1 above)
Comment (dpoltavets) [reply src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java]: Java embedded code has been removed

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/error/CacheException.java (hunk 2 above)
Comment (dpoltavets) [reply src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/error/CacheException.java]: This was needed for testing. This change has been reverted.

**Review by niksv — APPROVED** [review]

**Review by surabujin — APPROVED** [review]

**Pull request merged**